rasusa/0.7.0/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:focal as app
+
+ARG RASUSA_VER="0.7.0"
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="rasusa"
+LABEL software.version="0.7.0"
+LABEL description="Randomly subsample sequencing reads to a specified coverage."
+LABEL website="https://github.com/mbhall88/rasusa"
+LABEL license="https://github.com/mbhall88/rasusa/blob/master/LICENSE"
+LABEL maintainer1="Jake Garfin"
+LABEL maintainer1.email="jake.garfin@state.mn.us"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+
+# install wget and cleanup apt garbage
+RUN apt-get update && apt-get -y install --no-install-recommends \
+ wget \
+ ca-certificates && \
+ apt-get autoclean && \
+ rm -rf /var/lib/apt/lists/*
+
+# install rasusa
+RUN wget https://github.com/mbhall88/rasusa/releases/download/${RASUSA_VER}/rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+ tar -xvf rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+ rm -rf rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+ mkdir /data
+
+# set PATH and perl locale settings
+ENV PATH="${PATH}:/rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl/" \
+ LC_ALL=C
+
+WORKDIR /data
+
+FROM app as test 
+
+# print version and help options
+RUN rasusa --version && rasusa --help
+
+# cannot run rust-based test, since rust isn't installed, but just testing basic command on SC2 reads in our repo
```

Comment:
There seriously needs to be an `apt-get rust`

Reply:
agreed!